cli/pack/common.go

```diff
@@ -108,33 +111,61 @@ func detectVersion(ctx *context.Ctx) error {
 	return nil
 }
 
-func getPackageFullname(ctx *context.Ctx) string {
-	ext, found := extByType[ctx.Pack.Type]
-	if !found {
-		panic(project.InternalError("Unknown type: %s", ctx.Pack.Type))
+func getPackageSuffix(ctx *context.Ctx) string {
+	var suffix string
+	if ctx.Pack.Suffix != "" {
+		suffix = fmt.Sprintf("-%s", ctx.Pack.Suffix)
 	}
 
-	packageFullname := fmt.Sprintf(
-		"%s-%s",
+	return suffix
+}
+
+func getRpmPackageFullname(ctx *context.Ctx) string {
```

Comment:
I would like to see here a link to the document you lean on. The same for `getDebPackageFullname()`.

Reply:
Added